persist/fs/commitlog/options.go

```diff
@@ -88,6 +92,7 @@ func NewOptions() Options {
 		bytesPool: pool.NewCheckedBytesPool(nil, nil, func(s []pool.Bucket) pool.BytesPool {
 			return pool.NewBytesPool(s, nil)
 		}),
+		readConcurrency: defaultReadConcurrency,
```

Comment:
Could you add a check in `Validate()` to ensure this is `>0`

Reply:
Oh I didn't realize ya'll had a validate function for the options like that, yeah that sounds like a good idea.

Reply:
Sounds good